package.json

```diff
@@ -62,7 +61,7 @@
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-sonarjs": "^0.23.0",
     "eslint-plugin-unicorn": "^49.0.0",
-    "husky": "4.3.8",
+    "husky": "9.0.11",
```

Comment:
`husky` is deliberately pinned to 4.3.8. The package was rewritten entirely in a future version that made it much more difficult to use, IMO.

Reply:
check my comment: https://github.com/RetroAchievements/api-js/pull/94#discussion_r1605568539

Reply:
This implies our husky hooks need to be migrated. See: https://github.com/pnpm/pnpm/issues/4790#issuecomment-1136602175

Reply:
Done, added the husky hooks